cg/cli/store/store.py

```diff
@@ -16,15 +15,14 @@
 
 @click.group()
 @click.pass_context
-def store(context):
+def store(context: click.Context):
     """Command for storing files"""
     LOG.info("Running CG store command")
-    housekeeper_api = HousekeeperAPI(context.obj)
-    crunchy_api = CrunchyAPI(context.obj)
+    housekeeper_api = context.obj["housekeeper_api"]
```

Comment:
```suggestion
    housekeeper_api = context.obj.housekeeper_api
```
is this not intended to be this way? Dismiss if im wrong

Reply:
Correct! thx